packages/ember-simple-auth-cookie-store/lib/simple-auth-cookie-store/stores/cookie.js

```diff
@@ -96,6 +96,18 @@ export default Base.extend({
   _syncDataTimeout: null,
 
   /**
+    @property _renewExpirationTimeout
+    @private
+  */
+  _renewExpirationTimeout: null,
```

Comment:
I prefixed it with "_" because private _syncDataTimeout is prefixed with "_". Do you still want me to rename _renewExpirationTimeout to be renewExpirationTimeout ?